### PR TITLE
Enable test for nirvana

### PR DESCRIPTION
**Pull request by YangKeao**
### What problem does this PR solve?

Fix most tests for nirvana branch.

`go test -p 1 -coverprofile=/tmp/vscode-goNFVf7q/go-code-cover $(go list ./...|grep -vE "core|ptrace|vendor|apiserver|store")` this command will run all tests.

However, the `core` and `apiserver` and `store` still fail to compile.

### CODE REVIEW

File: controllers/podnetworkchaos/types_test.go [hunk 1]
```diff
@@ -15,20 +15,22 @@ package podnetworkchaos
...
 
 	. "github.com/onsi/gomega"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
```
[the remark this answers is not in the document]
Comment (YangKeao) [reply controllers/podnetworkchaos/types_test.go:27]: ignore

**Comment by STRRL** [comment]
How about enabling Github actions for the testcases?

**Comment by YangKeao** [comment]
> 
> 
> How about enabling Github actions for the testcases?

After merging the modification on apiserver and core, it could enable github actions for the testcases.

I will setup e2e test right now.

**Comment by YangKeao** [comment]
Oh. It fails to build `chaos-dashboard`.

**Pull request merged**